cms/djangoapps/contentstore/utils.py

```diff
@@ -88,13 +89,27 @@ def get_lms_link_for_item(location, preview=False):
     """
     assert isinstance(location, UsageKey)
 
-    if settings.LMS_BASE is None:
+    # checks LMS_BASE value in site configuration for the given course_org_filter(org)
+    # if not found returns settings.LMS_BASE
+    lms_base = SiteConfiguration.get_value_for_org(
+        location.org,
+        "LMS_BASE",
+        settings.LMS_BASE
+    )
+
+    # checks PREVIEW_LMS_BASE value in site configuration for the given course_org_filter(org)
+    # if not found returns settings.FEATURES.get('PREVIEW_LMS_BASE')
+    preview_base = SiteConfiguration.get_value_for_org(
+        location.org,
+        "PREVIEW_LMS_BASE",
+        settings.FEATURES.get('PREVIEW_LMS_BASE')
+    )
```

Comment:
We can remove this because we will look up preview base below only if we need to.